generator/views/funnel_analysis_view.py

```diff
@@ -104,19 +105,22 @@ def _funnel_analysis_lookml(self) -> List[Dict[str, Any]]:
                 "name": f"completed_step_{n}",
                 "type": "yesno",
                 "description": f"Whether the user completed step {n} on the associated day.",
-                "sql": "".join(
-                    (
-                        "REGEXP_CONTAINS(${TABLE}.events, mozfun.event_analysis.create_funnel_regex([",
-                        ",".join(
-                            [f"${{step_{ni}.match_string}}" for ni in range(1, n + 1)]
-                        ),
-                        "],",
-                        "True))",
-                    )
-                ),
+                "sql": dedent(
+f"""
+REGEXP_CONTAINS(
+    ${{TABLE}}.events, mozfun.event_analysis.create_funnel_regex(
+        [{", ".join([
+            f'${{step_{ni}.match_string}}' for ni in range(1, n + 1)
+        ])}],
+        True
+    )
+)
+"""
+                )
```

Comment:
Do you agree that this indenting is easier to read? And makes use of the `dedent` function
```suggestion
                "sql": dedent(f"""
                              REGEXP_CONTAINS(
                                  ${{TABLE}}.events, mozfun.event_analysis.create_funnel_regex(
                                      [{", ".join([
                                          f'${{step_{ni}.match_string}}' for ni in range(1, n + 1)
                                      ])}],
                                      True
                                  )
                              )
                              """)
```